mw-webapp/src/dataAccessLogic/useGetDataTableReports.ts

```diff
@@ -0,0 +1,25 @@
+import {useEffect, useState} from "react";
+import {getDayReports} from "src/dataAccessLogic/getDayReports";
+import {DayReport} from "src/model/businessModel/DayReport";
+
+/**
+ * Load day reports
+ * @returns {DayReport[]}
+ */
+export const useGetDataTableReports = () => {
+  const [data, setData] = useState<DayReport[]>([]);
```

Comment:
If you get DayReports data what do you thinkabout rename state to dayReports and setDayReports?

Reply:
done